src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -49,21 +52,39 @@ case class WorkflowActor(id: WorkflowId,
 
   when(WorkflowRunning) {
     case Event(CallActor.Started(call), NoFailureMessage) =>
+      log.info(s"$tag: call '${call.name}' started")
       storeActor ! UpdateStatus(call, ExecutionStatus.Running)
       stay()
     case Event(CallActor.Completed(completedCall, callOutputs), NoFailureMessage) =>
+      log.info(s"$tag: call '${completedCall.name}' completed")
       storeActor ! CallCompleted(completedCall, callOutputs)
       stay()
     case Event(StoreActor.RunnableCalls(runnableCalls), NoFailureMessage) =>
       if (runnableCalls.nonEmpty) {
-        log.info("Starting calls: " + runnableCalls.map {_.name}.toSeq.sorted.mkString(", "))
+        log.info(s"$tag: starting " + runnableCalls.map {_.name}.toSeq.sorted.mkString(", "))
       }
-      runnableCalls foreach startCallActor
+      runnableCalls foreach {call =>
+        log.info(s"$tag: call '${call.name}' -> Started")
+        storeActor ! UpdateStatusFrom(call, ExecutionStatus.NotStarted, ExecutionStatus.Started)
+      }
+      stay()
+    case Event(StoreActor.ExecutionStatusUpdate(call: Call, status: ExecutionStatus), NoFailureMessage) if status == ExecutionStatus.Started =>
+      log.info(s"$tag: starting call actor '${call.name}'")
+      val callActorProps = CallActor.props(call, id, backend, storeActor, "CallActor-" + call.name)
+      context.actorOf(callActorProps) ! CallActor.Start
+      stay()
+    case Event(StoreActor.ExecutionStatusUpdate(call: Call, status: ExecutionStatus), NoFailureMessage) =>
       stay()
     case Event(CallActor.Failed(call, failure), NoFailureMessage) =>
+      log.info(s"$tag: call '${call.name}' failed ($failure)")
       storeActor ! UpdateStatus(call, ExecutionStatus.Failed)
       goto(WorkflowFailed) using FailureMessage(failure)
-    case Event(WorkflowActor.Complete, NoFailureMessage) => goto(WorkflowSucceeded)
+    case Event(WorkflowActor.Complete, NoFailureMessage) =>
+      log.info(s"$tag: workflow complete")
+      goto(WorkflowSucceeded)
+    case x =>
```

Comment:
Oh right, that is correct.  I think I put this case here to debug something and forgot to take it out.